social_core/backends/open_id_connect.py

```diff
@@ -131,6 +139,53 @@ def auth_params(self, state=None):
         """Return extra arguments needed on auth process."""
         params = super().auth_params(state)
         params["nonce"] = self.get_and_store_nonce(self.authorization_url(), state)
+
+        display = self.setting("DISPLAY", default=self.DISPLAY)
+        if display is not None:
+            if not display:
+                raise ValueError("OpenID Connect display value cannot be empty string.")
```

Comment:
Maybe it would better to introduce a new exception in https://github.com/python-social-auth/social-core/blob/master/social_core/exceptions.py and use it instead (`AuthMissingParameter` is closest, but IMHO not really a good fit here).